Makefile.core.mk

```diff
@@ -27,7 +27,7 @@ export IN_BUILD_CONTAINER := $(IN_BUILD_CONTAINER)
 
 # ISTIO_IMAGE_VERSION stores the prefix used by default for the Docker images for Istio.
 # For example, a value of 1.6-alpha will assume a default TAG value of 1.6-dev.<SHA>
-ISTIO_IMAGE_VERSION ?= 1.15-alpha
+ISTIO_IMAGE_VERSION ?= 1.16-alpha
```

Comment:
THis change won't work as we are currently testing against the 1.15 release branch. If this change requires something in the master branch it will need to wait.

Reply:
Sure. I can wait for the 1.16 to merge this. This is to verify the test script in this PR so I bump this up temporarily